docs/en/setup/service-agent/java-agent/Application-toolkit-webflux.md

```diff
@@ -46,6 +52,28 @@
 ...
 ```
 
+### Fetch trace context relative IDs 
+```java
+    @Override
+    public Mono<Void> filter(ServerWebExchange exchange, GatewayFilterChain chain){
+        // fetch trace ID
+        String traceId = WebFluxSkyWalkingTraceContext.traceId(exchange);
```

Comment:
Please update all APIs you added. This is an official doc, we should provide a complete version.